src/tests/test_bootstrap.py

```diff
@@ -5,6 +5,7 @@
 import mock
 
 import requests_mock
+import builtins
```

Comment:
Could you please check the issue reported by Codacy (unused import)?

Reply:
Seems like Codacy can only analyse actual usages. This is used in the [mock](https://github.com/zalando-incubator/docker-locust/pull/87/files#diff-9c056b1bba9a05022f4af92ea0b8bb8eR67).. In python2, the builtins were available without an import but unfortunately that is not the case in py3. Not immediately obvious how to fix this. I'll try to see but is it possible to ignore as a last result?

Reply:
Ah, turns out that for the mock the import is not needed.